oauth2client/_helpers.py

```diff
@@ -11,12 +11,216 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 """Helper functions for commonly used utilities."""
 
 import base64
+import functools
+import inspect
 import json
+import logging
+import os
+import warnings
 
 import six
+from six.moves import urllib
+
+
+__author__ = [
+    'rafek@google.com (Rafe Kaplan)',
+    'guido@google.com (Guido van Rossum)',
+]
+
+__all__ = [
```

Comment:
Looks like no.  Is there any reason for us to define `__all__` here?  It looks like this file and `tools.py` are the only files that explicitly define `__all__`.  With this being a private module, I think it can be removed.